README.md

```diff
@@ -597,13 +598,27 @@ $response = $client->packages()->artifacts()->create($file, 'application/zip', $
 $artifactId = $response['id'];
 $client->packages()->createArtifactPackage([$artifactId]);
 ```
-#### Update artifact files of a package
+#### Add an artifact file to an existing package
 
 ```php
-$result = $client->packages()->artifacts()->showPackageArtifacts('acme-website/package'); // get artifact files details for a package
-$artifactFileIds = [42, 43];
-$client->packages()->editArtifactPackage('acme-website/package', $artifactFileIds);
+$packageName = 'acme/artifact';
+$fileName = 'package1.zip';
+$file = file_get_contents($fileName);
+$client->packages()->artifacts()->add($packageName, $file, 'application/zip', $fileName);
+```
+#### Update or replace artifact files of a package
+
+```php
+// in case you want to replace the artifact file with a newly uploaded one
+// 1. upload the new artifact file
+$fileName = 'package1.zip';
+$file = file_get_contents($fileName);
+$response = $client->packages()->artifacts()->create($file, 'application/zip', $fileName);
+$artifactId = $response['id'];
+// 2. send the artifact file ids you want to use for the package
+$client->packages()->editArtifactPackage('acme-website/package', [$artifactId, 42]);
```

Comment:
Looking at the example I do not see where the 42 is coming from. Can you make this a full example how you would replace a version including all the steps necessary?
* load the list of artifact files of a package
* remove the artifact that you don't want from the list
* upload a new file
* add the id to the list
* submit the package with editArtifactPackage